src/TopMenu/index.js

```diff
@@ -50,7 +52,7 @@ class TopMenu extends Component {
     return (
       <Wrapper>
         <Container
-          className="header"
```

Comment:
are there any css rules by chance behind this class?

Reply:
@shcheklein I have checked it. There are none. Only place the header class is used now is in the tooltip component to style the header inside the tooltip.